From 07d90cf77a33043482d01b720ec155b98f103cb4 Mon Sep 17 00:00:00 2001
From: Jackeagle <jackeagle102@gmail.com>
Date: Wed, 27 Mar 2019 06:11:54 +0100
Subject: [PATCH 14/15] Revert "vendor/bliss: Add separate init for
 BlissUpdater"

This reverts commit f814ada37fbdc0ff6cec776fb6328c5e1f8d4fdf.
---
 config/common.mk                         | 3 +--
 prebuilt/common/etc/init.bliss.rc        | 3 +++
 prebuilt/common/etc/init.blissupdater.rc | 3 ---
 3 files changed, 4 insertions(+), 5 deletions(-)
 delete mode 100644 prebuilt/common/etc/init.blissupdater.rc

diff --git a/config/common.mk b/config/common.mk
index 4a8c9fcca..5a5dee8ed 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -63,8 +63,7 @@ PRODUCT_COPY_FILES += \
 
 # Init file
 PRODUCT_COPY_FILES += \
-    vendor/bliss/prebuilt/common/etc/init.bliss.rc:root/init.bliss.rc \
-    vendor/bliss/prebuilt/common/etc/init.blissupdater.rc:system/etc/init/init.blissupdater.rc
+    vendor/bliss/prebuilt/common/etc/init.bliss.rc:root/init.bliss.rc
 
 # Bring in camera effects
 PRODUCT_COPY_FILES +=  \
diff --git a/prebuilt/common/etc/init.bliss.rc b/prebuilt/common/etc/init.bliss.rc
index d33d8474f..464d04c50 100644
--- a/prebuilt/common/etc/init.bliss.rc
+++ b/prebuilt/common/etc/init.bliss.rc
@@ -21,6 +21,9 @@ on init
 on post-fs-data
     mkdir /data/.ssh 0750 root shell
     
+    # Create an additional OTA package directory that unlike /data/bliss_updates
+    mkdir /data/bliss_updates 0770 system cache
+
     # Run sysinit
     start sysinit
 
diff --git a/prebuilt/common/etc/init.blissupdater.rc b/prebuilt/common/etc/init.blissupdater.rc
deleted file mode 100644
index ad991785b..000000000
--- a/prebuilt/common/etc/init.blissupdater.rc
+++ /dev/null
@@ -1,3 +0,0 @@
-on post-fs-data
-    # Create an additional OTA package directory that unlike /data/bliss_updates
-    mkdir /data/bliss_updates 0770 system cache
\ No newline at end of file
-- 
2.17.1

